docs/checks/internal/checks.yaml

```diff
@@ -581,8 +581,13 @@ checks:
       Signed releases attest to the provenance of the artifact.
 
       This check looks for the following filenames in the project's last five
-      releases: [*.minisig](https://github.com/jedisct1/minisign), *.asc (pgp),
-      *.sig, *.sign.
+      [release assets](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases):
+      [*.minisig](https://github.com/jedisct1/minisign), *.asc (pgp),
+      *.sig, *.sign, [*.intoto.jsonl](slsa.dev).
+
+      If a signature is found in the assets for each release, a score of 8 is given.
+      If a SLSA provenances is found in the assets for each release (*.intoto.jsonl), the maximum score of 10 is given.
```

Comment:
"If SLSA provenance" (singular, no "a") 
Or: "If a SLSA provenance file is found..."

Can you add a link to define SLSA provenance here, directly on the term rather than a few lines later? And link to a specific page, not just slsa.dev, so users don't have to search too hard.

Reply:
Done, thanks